components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -327,6 +353,20 @@ public void setOutboundProvisioningConfig(OutboundProvisioningConfig outboundPro
         this.outboundProvisioningConfig = outboundProvisioningConfig;
     }
 
+    /**
+     * @return
+     */
+    public AppRoleMappingConfig[] getApplicationRoleMappingConfig() {
+        return applicationRoleMappingConfig;
```

Comment:
add comments and new line
Check similar places

Reply:
Addressed in 3da9288685f309f6395405363622af44c39f34c2

Reply:
Add the method comment and add a new line before staring the method implementation.

Reply:
Addressed the missed places in eb2c437f152c27dff29633549de3b77457525dd4